lib/diagnostic.js

```diff
@@ -15,7 +15,13 @@ db.dropDatabase();
 // House Targaryen, motto 'Fire and Blood'
 
 // your code begins here
+mongoimport --db=mongo-crud --collection=houses --type=csv --headerline --file=data/houses.csv
 
+db.houses.insert({
```

Comment:
This will not run since the syntax is invalid (unmatched braces).  If you want to insert multiple documents at once, then they must be in an array.

```javascript
db.houses.insert([{
  name: 'House Arryn',
  motto: 'As High as Honor'
}, {
  name: 'House Stark',
  motto: 'Winter is Coming'
}, {
  name: 'House Targaryen',
  motto: 'Fire and Blood'
}]);
```